### PR TITLE
PPTP-1763 | Payment redirect link construction

### PR DESCRIPTION
**Pull request by liampan**
### Description of Work carried through

Brief description of what/why/how.

#### Check list 
 - [ ] `./precheck` was executed (Integration/Component/Unit tests)
 - [ ] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -67,6 +67,12 @@ microservice {
         url = "http://localhost:8504"
       }
 
+      pay-api {
```
[the remark this answers is not in the document]
Comment (liampan) [reply conf/application.conf:70]: still need to do this, will post pr here when done

File: app/views/IndexView.scala.html [hunk 2]
```diff
@@ -193,14 +200,17 @@ <h2 class="govuk-heading-l">@messages("account.homePage.title")</h2>
             links = Seq[Option[Html]](
                 returnsLink,
                 Some(link(id = "guidance-link", text = messages("account.homePage.card.makeReturn.guidance.link"), call = Call("GET", completeReturnUrl), newTab = true))
-            ).filter(_.isDefined).flatten,
+            ).filter(_.isDefined).flatten
         )
 
+        @* <!-- todo should the link always show? --> *@
```
Comment (liampan) [inline app/views/IndexView.scala.html:206]: think the answer to this is yes, there is no issue if the user follows this link even if amount is 0 more of a UCD question.

File: conf/application.conf (hunk 1 above)
[the remark this answers is not in the document]
Comment (liampan) [reply conf/application.conf:70]: https://github.com/hmrc/app-config-base/pull/7072

File: app/views/IndexView.scala.html (hunk 2 above)
Comment (lmcewanhmrc) [reply app/views/IndexView.scala.html:206]: I agree - allows user to still make a payment even if we're out of date and think they currently owe nothing

**Review by lmcewanhmrc — APPROVED** [review]

**Pull request merged**